lib/id3c/cli/command/etl/manifest.py

```diff
@@ -95,55 +95,72 @@ def etl_manifest(*, db: DatabaseSession):
         with db.savepoint(f"manifest record {manifest_record.id}"):
             LOG.info(f"Processing record {manifest_record.id}")
 
-            # Convert sample barcode to full identifier, ensuring it's
-            # known and from the correct identifier set.
-            sample_barcode = manifest_record.document.pop("sample")
-            sample_identifier = find_identifier(db, sample_barcode)
+            # Identify if reserved keys exist in the record
+            record_has_sample = "sample" in manifest_record.document
+            record_has_collection = "collection" in manifest_record.document
+            record_has_collected_date = "collected_date" in manifest_record.document
 
-            if not sample_identifier:
-                LOG.warning(f"Skipping sample with unknown sample barcode «{sample_barcode}»")
-                mark_skipped(db, manifest_record.id)
-                continue
+             # Sample collection date
+            collected_date = manifest_record.document.pop("collected_date", None)
```

Comment:
As discussed on Slack, I think we should make this backwards compatible so that we can re-process the existing records in `receiving.manifest` if needed.

Reply:
Thanks for the good discussion. I've changed the setting to "date_column" and the JSON key to "date" to be compatible with how things are done today.